pkg/kubelet/kubeletconfig/checkpoint/configmap.go

```diff
@@ -25,10 +25,9 @@ import (
 	kubeletconfig "k8s.io/kubernetes/pkg/kubelet/apis/kubelet.config.k8s.io"
 	kubeletscheme "k8s.io/kubernetes/pkg/kubelet/apis/kubelet.config.k8s.io/scheme"
 	utilcodec "k8s.io/kubernetes/pkg/kubelet/kubeletconfig/util/codec"
+	"k8s.io/kubernetes/pkg/kubelet/kubeletconfig/util/keys"
 )
 
-const configMapConfigKey = "kubelet"
```

Comment:
do we care about backwards compatibility of this?

Reply:
Not as long as dynamic kubelet configuration is an alpha feature. Which is still the case after this PR.